Core/Net/Repo.cs

```diff
@@ -131,7 +131,16 @@ internal static void UpdateRegistry(Uri repo, Registry registry, KSP ksp, IUser
         {
             log.InfoFormat("Downloading {0}", repo);
 
-            string repo_file = Net.Download(repo);
+            string repo_file = String.Empty;
```

Comment:
You are right, this isn't needed in csharp. That initialization should be removed.